apps/i18n/aichat/en_us.json

```diff
@@ -1,7 +1,21 @@
 {
+  "aichatWorkspaceDescription": "The area where students can interact directly with the chatbot they've created.",
   "aichatWorkspaceHeader": "AI Chat",
+  "chatComponentsHeader": "AI Chat Components",
+  "colonSpace": ": ",
   "continue": "Continue",
+  "instructions": "Instructions",
+  "instructionsDescription": "Clear directions and goals for each level.",
```

Comment:
It would be better to format the strings using markdown format so translators get full context: 
```

"teacherOnboardingModal_Instructions": "*Instructions*: Clear directions and goals for each level.",
```